### PR TITLE
Feature/IESP-70

### PR DESCRIPTION
**Pull request by melissa-dellorco**

### CODE REVIEW

File: src/main/java/it/pagopa/interop/probing/eservice/registry/updater/dao/EserviceDao.java [hunk 1]
```diff
@@ -33,75 +16,46 @@
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
 
-
 @Slf4j
 @Getter
 public class EserviceDao {
```
Comment (nttdata-rtorsoli) [inline src/main/java/it/pagopa/interop/probing/eservice/registry/updater/dao/EserviceDao.java:21]: Use Interface

File: src/main/java/it/pagopa/interop/probing/eservice/registry/updater/dao/EserviceDao.java [hunk 2]
```diff
@@ -33,75 +16,46 @@
...
 	private static final String PERSISTENCE_UNIT_NAME = "hibernate.persist.unit";
 
-	/** The Constant VERSION_ID_PARAM_NAME. */
 	private static final String VERSION_ID_PARAM_NAME = "versionIdParam";
 
-	/** The Constant SERVICE_ID_PARAM_NAME. */
 	private static final String SERVICE_ID_PARAM_NAME = "serviceIdParam";
 
-	/** The Constant FIND_BY_SERVICE_AND_VERSION_ID_QUERY. */
 	private static final String FIND_BY_SERVICE_AND_VERSION_ID_QUERY = "SELECT e FROM Eservice e WHERE e.eserviceId = :serviceIdParam AND e.versionId = :versionIdParam";
```
Comment (nttdata-rtorsoli) [inline src/main/java/it/pagopa/interop/probing/eservice/registry/updater/dao/EserviceDao.java:43]: No SQL directly

File: src/main/java/it/pagopa/interop/probing/eservice/registry/updater/dto/EserviceDTO.java [hunk 3]
```diff
@@ -40,37 +23,30 @@
 @ToString
 public class EserviceDTO {
```
Comment (nttdata-rtorsoli) [inline src/main/java/it/pagopa/interop/probing/eservice/registry/updater/dto/EserviceDTO.java:24]: Use interface

File: src/main/java/it/pagopa/interop/probing/eservice/registry/updater/model/Eservice.java [hunk 4]
```diff
@@ -49,57 +30,38 @@
...
- *
- * @return the java.lang. string
- */
 @Data
-
-/**
- * Instantiates a new eservice.
- */
 @NoArgsConstructor
 public class Eservice implements Serializable {
```
Comment (nttdata-rtorsoli) [inline src/main/java/it/pagopa/interop/probing/eservice/registry/updater/model/Eservice.java:38]: Use Interface and create an Abstract Entity to manage common fields and listener

File: src/main/java/it/pagopa/interop/probing/eservice/registry/updater/service/EserviceService.java [hunk 5]
```diff
@@ -29,33 +11,17 @@
 import it.pagopa.interop.probing.eservice.registry.updater.util.EserviceState;
 import it.pagopa.interop.probing.eservice.registry.updater.util.EserviceTechnology;
 
-/**
- * The Class EserviceService.
- */
 public class EserviceService {
```
Comment (nttdata-rtorsoli) [inline src/main/java/it/pagopa/interop/probing/eservice/registry/updater/service/EserviceService.java:14]: Use interface

**Review by nttdata-rtorsoli — APPROVED** [review]

File: src/main/java/it/pagopa/interop/probing/eservice/registry/updater/dto/EserviceDTO.java (hunk 3 above)
Comment (melissa-dellorco) [reply src/main/java/it/pagopa/interop/probing/eservice/registry/updater/dto/EserviceDTO.java:24]: this topic was already discussed in the previous pr

File: src/main/java/it/pagopa/interop/probing/eservice/registry/updater/service/EserviceService.java (hunk 5 above)
Comment (melissa-dellorco) [reply src/main/java/it/pagopa/interop/probing/eservice/registry/updater/service/EserviceService.java:14]: this topic was already discussed in the previous pr

File: src/main/java/it/pagopa/interop/probing/eservice/registry/updater/util/logging/LoggerLayout.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+	public String doLayout(ILoggingEvent event) {
+		StringBuilder sbuf = new StringBuilder();
+		sbuf.append(new Timestamp(event.getTimeStamp()));
+		sbuf.append(" ");
+		sbuf.append(event.getLevel());
+		sbuf.append(" [");
+		sbuf.append(event.getLoggerName());
+		sbuf.append("]");
+		try {
+			sbuf.append(" - [OID=");
```
Comment (galales) [inline src/main/java/it/pagopa/interop/probing/eservice/registry/updater/util/logging/LoggerLayout.java:24]:
```suggestion
			sbuf.append(" - [CID=");
```

**Review by galales — CHANGES_REQUESTED** [review]

File: src/main/java/it/pagopa/interop/probing/eservice/registry/updater/util/logging/LoggerLayout.java (hunk 6 above)
Comment (melissa-dellorco) [reply src/main/java/it/pagopa/interop/probing/eservice/registry/updater/util/logging/LoggerLayout.java]: done

**Review by nttdata-rtorsoli — APPROVED** [review]

**Review by galales — APPROVED** [review]

**Review by galales — APPROVED** [review]

**Review by nttdata-rtorsoli — APPROVED** [review]

**Pull request merged**